package.json

```diff
@@ -1,53 +1,34 @@
 {
   "name": "@sajari/sdk-node",
-  "version": "2.0.0-beta.7",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
-  "author": "Ben Hinchley <bhinchley@sajari.com>",
+  "version": "4.0.0-alpha.1",
+  "repository": "git@github.com:github.com/sajari/sdk-node.git",
+  "author": "Sajari Pty Ltd",
   "license": "MIT",
+  "main": "./build/src/index.js",
```

Comment:
Is it worth us adding ESM support at this stage while we're doing a big change?
https://github.com/standard-things/esm

Reply:
Yep. Just need to get the MVP out though to begin with!

Reply:
Cool, yeah I guess it might be nice to do in `4.1.0`?

Reply:
Yeah even 4.0 might be possible. I'm only looking to get a v4 alpha or beta tag out so there may be time to do it anyway.